easybuild/easyconfigs/m/MACS3/MACS3-3.0.0b2-foss-2021b.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'MACS3'
+version = '3.0.0b2'
+
+homepage = 'https://macs3-project.github.io/MACS/'
+description = """Model Based Analysis for ChIP-Seq data"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+
+builddependencies = [
+    ('pybind11', '2.7.1'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('scikit-learn', '1.0.1'),
+    ('matplotlib', '3.4.3'),
+]
+
+use_pip = True
+
```

Comment:
@fizwit please add the missing `sanity_pip_check`.

Reply:
MACS3 requires hmmlearn 0.3.0, which requires a new pybind11.  Need to switch to a newer toolchain.